src/aa_remove_data/remove_data.py

```diff
@@ -0,0 +1,486 @@
+import argparse
+import time
+from collections.abc import Callable
+from datetime import datetime
+from pathlib import Path
+from typing import Any
+
+from aa_remove_data.pb_utils import DEFAULT_CHUNK_SIZE, PBUtils
+
+
+def get_nano_diff(sample1: type, sample2: type) -> int:
```

Comment:
type: this looks like a strange data type to have as an input. Do you know what it is? I'm not so familiar myself

Reply:
Resolved to use the Any type in place of type